Library/ViewModels/DiscoveryFiltersViewModel.swift

```diff
@@ -215,9 +214,17 @@ public final class DiscoveryFiltersViewModel: DiscoveryFiltersViewModelType,
 private func expandableRows(selectedRow: SelectableRow,
                             categories: [KsApi.Category]) -> [ExpandableRow] {
 
-  let expandableRows = categories
+  var grouped: [KsApi.Category:[KsApi.Category]] = [:]
```

Comment:
I think we could satisfy the compiler here and lose the `var` and `forEach`:

```swift
let expandableRows = categories
    .sorted { lhs, _ in !lhs.isRoot }
    .groupedBy { category -> KsApi.Category in
      guard let parent = category.parent else { return category }
      return parent
    }
```

Seems to build for me 👍